src/CSET/operators/regrid.py

```diff
@@ -319,5 +326,41 @@ def regrid_to_single_point(
     regrid_method = getattr(iris.analysis, method, None)
     if not callable(regrid_method):
         raise NotImplementedError(f"Does not currently support {method} regrid method")
-    cube_rgd = cube.interpolate(((lat, lat_pt), (lon, lon_pt)), regrid_method())
+    cube_rgd = cube.interpolate(((lat, lat_tr), (lon, lon_tr)), regrid_method())
     return cube_rgd
+
+
+def transform_lat_long_points(lon, lat, cube):
+    """Transform a selected point in longitude and latitude in ...
+
+    the real world to the corresponding point on the rotated grid
+    of a cube.
```

Comment:
Its best not to break the first line.

Generally the first line acts as a subject line, or a very concise summary. Then you can go into a bit more detail in however many paragraphs you need after the line break.